fastserde/avro-fastserde/src/main/java/com/linkedin/avro/fastserde/FastDeserializerGenerator.java

```diff
@@ -1148,21 +1163,26 @@ private void processEnum(final Schema schema, final JBlock body, FieldAction act
     }
   }
 
-  private void processBytes(JBlock body, FieldAction action, BiConsumer<JBlock, JExpression> putValueIntoParent,
-      Supplier<JExpression> reuseSupplier) {
+  private void processBytes(Schema schema, JBlock body, FieldAction action,
```

Comment:
And what about converting to logicalTypes from FIXED bytes?

Reply:
Ah, right.
I was a bit confused because **vanilla Avro does not provide any default converter for `Duration`** which is based on `fixed` type and I forgot to revisit `processFixed()` method.
I'll try to add it as part of custom-converter testing (coming soon).

Reply:
Support added in commit `[review-fixes] Part 7: Type 'fixed' now supports custom logical types.`

Due to limitation of current (Gradle) build scripts I created separate projects to confirm it works:
- [.avsc file](https://github.com/krisso-rtb/avro-custom-logical-types/blob/master/custom-logical-types-used/src/main/resources/schema/record-with-custom-logical-types.avsc)
- [unit test](https://github.com/krisso-rtb/avro-custom-logical-types/blob/master/custom-logical-types-used/src/test/java/com/rtbhouse/custom/logical/types/CustomLogicalTypesTest.java)